node/src/dev_service.rs

```diff
@@ -252,6 +276,13 @@ pub fn new_full(
 		config,
 	})?;
 
+	// Spawn trace_filter cache task if enabled.
+	if let Some(trace_filter_task) = trace_filter_task {
+		task_manager
+			.spawn_essential_handle()
+			.spawn("trace-filter-cache", trace_filter_task);
```

Comment:
Is this background task meant to mantain the expired cache (aka `blocks_to_check`) but that is still TODO?

Reply:
The background task is meant to manage the cache, from checking cache hit or miss, filling it and clearing it.
It's complete but cache expiration should be tested somehow (custom RPC endpoint to check cache hit/miss ?).